output/v0.0.json

```diff
@@ -3376,6 +3376,21 @@
                         "readOnly": false,
                         "maxLength": 250,
                         "description": "This sensitive information is stored encrypted and can only be retrieved through the API."
+                    },
+                    "payment_group_id": {
+                        "type": "string",
+                        "description": "",
```

Comment:
Optional value that will be used to group invoices into payments. When specified, only invoices with the same value will be included in a single payment.